scheduler/reconcile_test.go

```diff
@@ -345,7 +345,7 @@ func TestReconciler_Place_NoExisting(t *testing.T) {
 	job := mock.Job()
 	reconciler := NewAllocReconciler(
 		testlog.HCLogger(t), allocUpdateFnIgnore, false, job.ID, job,
-		nil, nil, nil, "", job.Priority)
+		nil, nil, nil, "", job.Priority, false)
```

Comment:
I would set all of these values to `true` in this test file so that we're not hiding any potential bugs. If you want to have a test that checks the code path, that's reasonable (maybe even tear it out after this release), but most tests should default to having the new always-on feature set to on.

Reply:
Fixed in latest push.